xc/xenops_server_xen.ml

```diff
@@ -1903,6 +1903,7 @@ module VGPU = struct
 				let emulator_pid =
 					match vgpu.implementation with
 					| GVT_g _ -> Device.Qemu.pid ~xs frontend_domid
+					| MxGPU _
```

Comment:
Actually, we have a slightly odd situation here, since we are using both a PCI device and a VGPU "device" for MxGPU. The `get_state` calls are used in the `update_pci` and `update_vgpu` functions in Xapi_xenops, which both drive the `VGPU.currently_attached` field. So far, at most one of these two `update_` functions would be called per VM, but for the new case both will be called, and both will attempt to set `VGPU.currently_attached`. It's really the PCI state that matters for MxGPU, so we probably should make `update_vgpu` not set `VGPU.currently_attached` for MxGPU.